Leanplum-SDK/Classes/Managers/Networking/LPNetworkFactory.m

```diff
@@ -25,41 +25,23 @@
 #import "LPNetworkFactory.h"
 #import "LPNetworkEngine.h"
 #import "LPNetworkOperation.h"
-#import "Leanplum_MKNKOperationWrapper.h"
```

Comment:
Does it mean we no longer need Leanplum_MKNKOperationWrapper.h all together? Since its only used below ios 7??

Reply:
yes

Reply:
Can we just remove entire Leanplum_MKNK related files?

Reply:
those are actually being removed. LMK if there are files that you see that are not being removed